### PR TITLE
fr-FR: stepping down as maintainer

### PR DESCRIPTION
**Pull request by JoelTroch**
Hello,

It's been a very long time since I have not contributed to the french translation of OpenRCT2 and that is because my free time these days is getting shorter and I have to focus on other important things.

For these reasons, I prefer to step down as the "french localization maintainer" by "removing myself" from the notifications in issue templates and my name in the maintainer list. My place would be better suited for someone like @rmnvgr who contributed a lot these days by adding the missing strings but most importantly: fix the existing translation. Something I wanted to do months ago but time never allowed me to focus on that.

Thank you for allowing me to contribute to this amazing project. Translating OpenRCT2 and programming the initial [keyboard shortcuts for track construction](https://github.com/OpenRCT2/OpenRCT2/pull/5200) support were fun experiences that I will always remember.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +64,7 @@ To do this, go to the directory where OpenRCT2 resides (not to be confused with
 | [![](https://img.shields.io/badge/zh--CN-maintained-green.svg)](https://github.com/OpenRCT2/Localisation/blob/master/data/language/zh-CN.txt) | [izhangfei](https://github.com/izhangfei)          |
 | [![](https://img.shields.io/badge/zh--TW-maintained-green.svg)](https://github.com/OpenRCT2/Localisation/blob/master/data/language/zh-TW.txt) | [daihakken](https://github.com/daihakken)         |
 | Not maintained <!-- Languages that are outdated with strings missing from OpenRCT2/vanilla-->| |                                                 
+| [![](https://img.shields.io/badge/fr--FR-outdated-yellow.svg)](https://github.com/OpenRCT2/Localisation/blob/master/data/language/fr-FR.txt)  |                                                    |
```
Comment (tupaschoal) [inline README.md:67]: I'm wondering whether it should be moved here, as it isn't really outdated, just won't have any "assigned" maintainer. What do you think @Gymnasiast ?

**Review by tupaschoal — COMMENTED** [review]
I'm glad it was a great experience for you. It's sad to see you part ways, but I can understand your reasons, feel free to drop by every now and then if you ever want to!

File: README.md (hunk 1 above)
Comment (Gymnasiast) [reply README.md:67]: I agree, just leave it in the same place and only remove your username.

**Comment by JoelTroch** [comment]
Requested changes have been applied

**Comment by Gymnasiast** [comment]
Thanks for all your contributions!

**Pull request merged**